draft-packages/empty-state/KaizenDraft/EmptyState/responsive.scss

```diff
@@ -1,34 +1,43 @@
-@import "~@kaizen/deprecated-component-library-helpers/styles/layout";
 @import "~@kaizen/design-tokens/sass/layout";
 
 $small: 300px;
 $medium: $layout-breakpoints-medium;
 $large: $layout-breakpoints-large;
 
+$side-padding: 48px;
```

Comment:
There are already some existing `px` values in here, but I feel we should only add `rem` values, if we can - plus change the old `px` ones if we can easily test it causes no regressions.
This will help when users zoom in/out.

Reply:
I was on the fence whether I should change this to a rem value, it could be supplanted by a spacing-xl but given it was only used in the media query below which uses a pixel value from design tokens I opted to leave it as is so as to not worry about calculating pixels and rems together.

Reply:
I don't have strong feelings about either way here at this point. We as a team should probably align on what to use (and I'd say it should be `rems`). Don't have the link handy but you might remember the post I linked about which value is best for accessibility and how it behaves.

Reply:
It's worth looking at how the component behaves when you change the browser text size and using that to make a judgement call on whether the padding should grow/shrink with font size or not.

Reply:
@christian314159 That's fair, I think in this instance it since its replacing the a pixel value calculated off $ca-layout-side-padding we should leave it as is. On Kevins recommendation I tried out it out with resized text and it didn't have any unfavourable impacts on the layout. I've added a small comment in the scss just to mark where the value comes from.